src/lib/Hydra/Helper/AddBuilds.pm

```diff
@@ -297,6 +297,24 @@ sub inputsToArgs {
             if ($alt->{type} eq "string") {
                 push @res, "--argstr", $input, $alt->{value};
             }
+            elsif ($alt->{type} eq "githubAPI") {
+                my $template = "githubAPIXXXXX";
+                my ($fh, $filename) = File::Temp::tempfile( $template, TMPDIR => 1, SUFFIX => '.json');
+                print $fh $alt->{value};
+                close $fh;
+
+                # Add file to store
+                my $adder = "nix-store";
+                my @cmd = ($adder, "--add", $filename);
+                (my $res, my $storePath, my $stderr) = captureStdoutStderr(21600, @cmd);
+                die "$adder returned " . ($res & 127 ? "signal $res" : "exit code " . ($res >> 8))
+                    . " while adding github input to store:\n" . ($stderr ? decode("utf-8", $stderr) : "(no output)\n")
+                    if $res;
+                unlink $filename or print STDERR "WARNING: unable to remove temporary file $filename\n";
+
+                $storePath =~ s/^\s+|\s+$//g;
+                push @res, "--arg", $input, "builtins.readFile $storePath";
```

Comment:
Can we make this `builtins.fromJSON (builtins.readFile $storePath)`? The nixexpr shouldn't care that the data it's passed was serialized as JSON at some point.